src/main/java/picard/analysis/directed/RnaSeqMetricsCollector.java

```diff
@@ -95,21 +95,34 @@ public RnaSeqMetricsCollector(final Set<MetricAccumulationLevel> accumulationLev
         return ignoredSequenceIndices;
     }
 
-    private class PerUnitRnaSeqMetricsCollector implements PerUnitMetricCollector<RnaSeqMetrics, Integer, SAMRecord> {
+    protected class PerUnitRnaSeqMetricsCollector implements PerUnitMetricCollector<RnaSeqMetrics, Integer, SAMRecord> {
+
+        protected final RnaSeqMetrics metrics;
```

Comment:
hmmm. It's nice to have a final metrics object, no? perhaps this can be done with a getMetrics that is overrided by the inherited class.